README.md

```diff
@@ -0,0 +1,183 @@
+| :woman_technologist: **[User Guides]**      | How to use sktime and its features.                             |
+| :scissors: **[Extension Templates]** | How to build your own estimator using sktime's API.            |
+| :control_knobs: **[API Reference]**      | The detailed reference for sktime's API.                        |
+| :tv: **[Video Tutorial]**            | Our video tutorial from the 2020 PyData Festival.      |
+| :hammer_and_wrench: **[Changelog]**          | Changes and version history.                                   |
+| :deciduous_tree: **[Roadmap]**          | sktime's software and community development plan.                                   |
+| :pencil: **[Related Software]**          | A list of related software. |
+
+[tutorials]: https://www.sktime.org/en/latest/tutorials.html
+[user guides]: https://www.sktime.org/en/latest/user_guide.html
+[video tutorial]: https://github.com/sktime/sktime-tutorial-pydata-amsterdam-2020
+[api reference]: https://www.sktime.org/en/latest/api_reference.html
+[changelog]: https://www.sktime.org/en/latest/changelog.html
+[roadmap]: https://www.sktime.org/en/latest/roadmap.html
+[related software]: https://www.sktime.org/en/latest/related_software.html
+
+## :speech_balloon: Where to ask questions
+
+Questions and feedback are extremely welcome! Please understand that we won't be able to provide individual support via email. We also believe that help is much more valuable if it's shared publicly, so that more people can benefit from it.
+
+| Type                            | Platforms                               |
+| ------------------------------- | --------------------------------------- |
+| :bug: **Bug Reports**              | [GitHub Issue Tracker]                  |
+| :sparkles: **Feature Requests & Ideas** | [GitHub Issue Tracker]                       |
+| :woman_technologist: **Usage Questions**          | [GitHub Discussions] · [Stack Overflow] |
+| :speech_balloon: **General Discussion**        | [GitHub Discussions] · [Gitter] · [Discord]          |
+
+[github issue tracker]: https://github.com/alan-turing-institute/sktime/issues
+[github discussions]: https://github.com/alan-turing-institute/sktime/discussions
+[stack overflow]: https://stackoverflow.com/questions/tagged/sktime
+[gitter]: https://gitter.im/sktime/community
+[discord]: https://discord.com/invite/gqSab2K
+
+
+## :stars: Features
+sktime provides a __unified interface for distinct but related time series learning tasks__. It features __dedicated time series algorithms__ and __tools for composite model building__ including pipelining, ensembling, tuning and reduction that enables users to apply an algorithm for one task to another.
+
+sktime also provides **interfaces to related libraries**, for example [scikit-learn], [statsmodels], [tsfresh], [PyOD] and [fbprophet], among others. Our aim is to make the time series analysis ecosystem more interoperable and usable as a whole.
```

Comment:
I think it makes sense to have `Our aim is to make the time series analysis ecosystem more interoperable and usable as a whole.` as an opening statement in the beginning of `Features`.